selenium/Tests/EditGraphTests.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System.Collections.Generic;
+using OpenQA.Selenium;
+using System;
+using System.Windows.Forms;
+
+namespace SeleniumTests.Tests
+{
+    class EditGraphTests
```

Comment:
These test classes contain `public static` members only. Consider making the classes themselves `public static` as well.